package.json

```diff
@@ -22,7 +22,8 @@
     "files": [
       "build/**/*",
       "main/dist.js",
-      "!node_modules/**/*"
+      "!node_modules/**/*",
+      "./assets/*"
```

Comment:
I currently also store the source files of the icon and other stuff in the assets folder and would not want to have them added to the bundle. I'd rather whitelist the exact assets that should be added.